app/src/main/java/com/github/dfa/diaspora_android/activity/AboutActivity.java

```diff
@@ -169,12 +184,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                     e.printStackTrace();
                 }
             }
+            contributeBtn.setOnClickListener(this);
```

Comment:
Could all be done by using
```
@OnClick( {R.id.id1, R.id.id2, R.id.id3} )
public void onSomethingClicked(View view){

}
```
Then also no Interface implementation is needed :D

And also: If nothing but the onClick event is needed, the `@BindViews` can even be removed :D